app/Builder/DockerComposeRun.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Builder;
+
+use App\Builder\Concerns\HasEnvironmentVariables;
+
+class DockerComposeRun extends Builder
+{
+    use HasEnvironmentVariables;
+
+    /** @var string $user */
+    protected $user = '';
+
+    public function getProgramName() : string
+    {
+        return 'run';
```

Comment:
this is the very same code as `DockerComposeExec`,  the only difference is this programname, maybe worth creating and abstract `DockerComposeBase`? 

Not sure if worth it, just wondering

Reply:
Not only this but the env variable also